manifest.yaml

```diff
@@ -117,6 +117,15 @@ postprocess:
      grep -q '# RHEL-only: Disable /tmp on tmpfs' /usr/lib/systemd/system/basic.target
      echo '# RHCOS-only: we follow the Fedora/upstream default' >> /usr/lib/systemd/system/basic.target
      echo 'Wants=tmp.mount' >> /usr/lib/systemd/system/basic.target
+
+     # TEMPORARY: Remove `ipmi-si` in /usr/lib/modules-load.d/fwupd-redfish.conf until fix is landed to RHEL 8.6
+     # Fixed version is fwupd-1.7.4-1.el8
+     # See https://github.com/openshift/os/issues/749
+     #     https://bugzilla.redhat.com/show_bug.cgi?id=2037294
+     #     https://github.com/fwupd/fwupd/pull/4162
+     [ ! -f /usr/lib/modules-load.d/fwupd-redfish.conf ] && exit 0
+     grep -q ipmi-si /usr/lib/modules-load.d/fwupd-redfish.conf || exit 100
```

Comment:
I'm guessing someone else asked for this but IMO, the "hard fail the build when the workaround is no longer needed" causes more pain than it solves.  I think it's nicer to ourselves to do "tech debt reminders", e.g. scheduling a reminder for 3 months later to check and see if a workaround is still needed.

(But, this is fine as is)

Reply:
Good suggestion. I should add more text to clarify when to remove this temporary workaround in the future, for this maybe "Should remove this workaround if the version of fwupd >= 1.7.4-1.el8"

Reply:
I added a reminder on our internal Slack instance to remove this when 8.6 hits GA; presumably that is when the fixed `fwupd` will land